tests/benchmarks/graph500-scale18-ef16_1hop.yml

```diff
@@ -0,0 +1,27 @@
+name: "GRAPH500-SCALE_18-EF_16-1_HOP"
+description: "Dataset: Synthetic graph500 network of scale 18 (262144x262144, 4194304 edges)
+                       - 262017 nodes with label 'Node'
+                       - 4194304 relations of type 'IS_CONNECTED'
+                       - Indexed properties: 
+                          - exact-match: IS_CONNECTED; [external_id]
```

Comment:
I'm not certain of the appropriate syntax for this line, but it seems likely to me that we should have `Node` instead of `IS_CONNECTED` (the relation type).

This comment holds for all the introduced files.